docs/09_get-involved/index.md

```diff
@@ -1,10 +1,73 @@
-# Get Involved with EOSIO Documentation!
+We appreciate your interest in contributing to the EOSIO platform! We always welcome contributions from our community to make our code and docs better. 
 
-We appreciate your interest in contributing to the EOSIO platform documentation! We always welcome contributions from our community to make our docs better.
+* [Get Involved with EOSIO Community and Code](#get-involved-with-eosio-code)
+* [Get Involved with EOSIO Documentation](#get-involved-with-eosio-documentation)
 
-## How to Contribute
+## Get Involved with EOSIO Community and Code
 
-You can either submit a Pull Request for quick edits or file a new docs issue.
+The following are different ways you can get involved with the EOSIO developers community and EOSIO repositories, which are hosted in the [EOSIO](https://github.com/EOSIO) Organization on GitHub. 
+
+
+
+### Developers Community Involvement
+
+#### EOSIO Stack Exchange
+Engage the EOSIO developer community over [EOSIO Stack Exchange](https://eosio.stackexchange.com/). You can ask a question related to EOSIO or answer questions and share your EOSIO knowledge with the community. 
+
+#### EOSIO Developer Telegram
+Instantly engage the EOSIO developer community over [EOSIO Telegram](https://t.me/joinchat/EaEnSUPktgfoI-XPfMYtcQ) instant messenger. Get involved by sharing your expertise in EOSIO development discussions, sharing community tools and projects that can assist EOSIO developers, and more. 
+
+
+### Get Involved with EOSIO Code
+
+EOSIO is a large open source project with over 500 repositories for its stack and core components. Some of the crucial repositories are but not limited to:
+
+* [eos](https://github.com/EOSIO/eos)
```

Comment:
Don't reference these specific repos, link to `software manuals` instead.

Reply:
and link to github organization http://github.com/EOSIO

Reply:
Removed references to specific EOSIO repos and also added EOSIO Org link. 

The text talks about code contributions. Do we need to link to `software manuals?`